src/pydocstyle/config.py

```diff
@@ -518,12 +525,20 @@ def _create_option_parser(cls):
                      "matches all dirs that don't start with "
                      "a dot").format(cls.DEFAULT_MATCH_DIR_RE))
 
+        # Decorators
+        option('--ignore-decorators', metavar='<decorators>', default=None,
```

Comment:
I wonder, should there be special handling of "qualified" decorators? e.g., `@wraps` vs `@functools.wraps`.